lib/fs/reader_at.go

```diff
@@ -242,11 +242,11 @@ func mmapFile(f *os.File, size int64) ([]byte, error) {
 	}
 	data, err := mmap(int(f.Fd()), int(size))
 	if err != nil {
-		return nil, fmt.Errorf("cannot mmap file with size %d bytes; already mmaped files: %d: %w; "+
-			"try increasing /proc/sys/vm/max_map_count or passing -fs.disableMmap command-line flag to the application", size, mmapedFiles.Get(), err)
+		return nil, fmt.Errorf("cannot mmap file with size %d bytes; already memory mapped files: %d: %w; "+
+			"try increasing /proc/sys/vm/max_map_count or passing -fs.disableMmap command-line flag to the application", size, mmappedFiles.Get(), err)
 	}
-	mmapedFiles.Inc()
+	mmappedFiles.Inc()
 	return data[:sizeOrig], nil
 }
 
-var mmapedFiles = metrics.NewCounter("vm_mmaped_files")
+var mmappedFiles = metrics.NewCounter("vm_mmapped_files")
```

Comment:
Should we mention this change in changelog?

Reply:
just added